src/components/Todo.jsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+import plus from "../assets/plus.svg";
+import Task from "./Task";
+import { nanoid } from "nanoid";
+
+export const Todo = () => {
+  const [text, setText] = useState({ taskName: "" });
+  const [tasks, setTasks] = useState([]);
+
+  const handleText = (e) => {
+    const { name, value } = e.target;
+    setText((prev) => {
+      return {
+        ...prev,
+        [name]: value,
+      };
+    });
+  };
+  const createNote = () => {
+    if (text.taskName === "") {
+      return alert("Note name must be provided");
+    }
+    const task = {
+      id: nanoid(),
+      value: text.taskName,
+      checked: false,
+    };
+    setTasks((oldTask) => [...oldTask, task]);
+    setText({ taskName: "" });
+  };
+  const handleClick = (id) => {
+    const currentTask = tasks.map((el) =>
```

Comment:
Could you use a more descriptive name for the `el` argument?

Reply:
sure